pkgs/tools/security/gnupg/21.nix

```diff
@@ -29,7 +29,12 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
 
-  patches = [ ./fix-libusb-include-path.patch ];
+  patches = [
+    ./fix-libusb-include-path.patch
+    # TODO: Remove the following patches on the next gnupg release after 2.1.18
+    ./0028-scd-Backport-two-fixes-from-master.patch
+    ./0029-scd-Fix-use-case-of-PC-SC.patch
```

Comment:
@fpletz  While I can find the "Fix-use-case-of-PC-SC" commit in the gnupg repo, I can't find the "Backport two fixes from master" patch in the gnupg repo---probably because the changes in that patch were spread across other commits in master.  I've opted to fetch these patches from a debian source. Let me know if you'd prefer something else.